beancount_import/source/schwab_csv_test.py

```diff
@@ -21,8 +21,8 @@ def test_source(name: str):
         example_dir=example_dir,
         source_spec={
             'module': 'beancount_import.source.schwab_csv',
-            "transaction_csv_filenames": glob.glob(f"{example_dir}/transactions/*.CSV"),
-            "position_csv_filenames": glob.glob(f"{example_dir}/positions/*.CSV"),
+            "transaction_csv_filenames": sorted(glob.glob(f"{example_dir}/transactions/*.CSV")),
```

Comment:
You can rebase your branch on latest master with #79 merged to eliminate this change from this PR.

Reply:
Done.

Reply:
I did rebase just now before pushing the latest changes. Not sure why this is still showing up.

Reply:
Hmm, doesn't seem to have pushed with the rebase, since I still see these changes here.

Reply:
Solved now.